lib/tutorials/en_US/getting-started.md

```diff
@@ -18,23 +19,26 @@ The most basic server looks like the following:
 
 const Hapi = require('hapi');
 
-const server = new Hapi.Server();
-server.connection({ port: 3000, host: 'localhost' });
+const server = Hapi.server({ port: 3000, host: 'localhost' });
 
-server.start((err) => {
-
-    if (err) {
-        throw err;
+const init = async () => {
+    try {
+        await server.start();
+        console.log(`Server running at: ${server.info.uri}`);
+    } catch (err) {
+        console.error(err);
     }
-    console.log(`Server running at: ${server.info.uri}`);
-});
+}
+
+init();
```

Comment:
we need to come up with a consistent pattern for this init method.. some of our repos have an example where `init` doesn't have a try/catch, and instead we put a `.catch()` on the call to `init`, here we're putting the try/catch in the `init` function itself. we should likely also be explicitly calling `process.exit(1)` in the catch, since an unhandled promise rejection won't necessarily exit the process. maybe this needs to be a style rule. @hueniverse?

Reply:
Not sure about a style rule. I would rather listen to unhandled rejections and exit there and wrap the rest of the async calls in a `start()` function then call it.